packages/salesforcedx-vscode-apex-debugger/src/messages/i18n.ts

```diff
@@ -16,8 +16,14 @@
  * If ommitted, we will assume _message.
  */
 export const messages = {
+  command_error_help_text: 'Command returned the following error',
```

Comment:
@DatGuyJonathan - How will these messages be displayed? Will there be some sort of punctuation between the `command_error_help_text`/`command_output_help_text`/`streaming_handshake_error_text` message and the error or output that follows it?